scripts/_lib.sh

```diff
@@ -1,4 +1,30 @@
-npmCommand="yarn"
+npmCommand="npm"
+babelCliVersion="^6.18.0"
+
+function initialize {
+  path=${1-.}
+
+  # Install yarn to Meteor globals. Enable `meteor yarn` subcommand.
```

Comment:
https://github.com/meteor/meteor/issues/7918#issuecomment-256179094

The idea is to enable to Meteor automatically. I guess until they include it in Meteor, we must do in that way.

 Moreover, with the `p-install` script happening also when `npm install`, it will install dependencies with more priority by using the `meteor yarn` command, that way we ensure consistency with Meteor node version, there are annoying issue happening around that and force us to set it up on our env.

In fact I would add two options only then, either `meteor npm` or `meteor yarn` possibilities, and remove the others (`npm` and `yarn`).